src/event.js

```diff
@@ -120,6 +120,11 @@ jQuery.event = {
 			selector = handleObjIn.selector;
 		}
 
+		// If the selector is invalid, throw any exceptions at attach time
+		if ( selector ) {
+			jQuery.find( selector, elem );
```

Comment:
None that I can think of.  The only case I can think of that breaks is if a selector extension was added after the event was attached here, but before an event was fired. That's probably not common?